pyproject.toml

```diff
@@ -15,6 +15,11 @@ ignore_missing_imports = true
 
 # refer to: https://docs.pytest.org/en/stable/mark.html
 [tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "--cov-report term-missing --cov=src/"
+testpaths = [
+    "tests",
+]
```

Comment:
I‘m wondering where can I see the report? If it can ,can you try to add the report tag to the readme file?  

refer to:
* https://github.com/nedbat/coveragepy
* https://github.com/marketplace/codecov

Reply:
I've tried  in my fork repo, it's all ok. You can see https://github.com/helloocc/python-wechaty/pull/1

In the repo, it needs the owner to sign up in [https://app.codecov.io/gh](https://app.codecov.io/gh) first. 
Then, with this pr, you can see the coverage report in comment，which also can be seen in the workflow log.
After that, put the coverage link in README header will be ok.